docs/README.md

```diff
@@ -8,6 +8,10 @@
 
 - [Development instructions](development.md)
 
+## Architecture
+
+- [Design](design.md)
+
 ## Project Documentation
```

Comment:
Let's remove this section. I think it's implied that all of this is project documentation.

Reply:
moved under Project documentation